### PR TITLE
Add svg object to configuration

### PR DESCRIPTION
**Pull request by jacobarriola**
The compiled sprite does not have style and dimension attributes in the <svg> root element so that it can be embedded in a project document without affecting layout (absolute positioning). 

To fix, added the svg object that allows authors to set attributes. Adding width, height and absolute positioning allows us to embed the inline sprite in our project html without affecting other surrounding elements. :art: :smile: 

Also removed the XML declaration and DOCTYPE declaration because it doesn't seem necessary to have. At worst, it gives authors the knowledge of being able to include or not include these elements in thier final compiled svg element. :smile:

### CODE REVIEW

File: Gulpfile.js [hunk 1]
```diff
@@ -16,6 +16,15 @@ var config = {
       sprite: 'sprite.svg', //sprite name
       example: true // Build sample page
     }
+  },
+  svg: {
+    xmlDeclaration: false, // strip out the XML attribute
+    doctypeDeclaration: false, // don't include the !DOCTYPE declaration
```
Comment (una) [inline Gulpfile.js]: stripping these are great additions -- I'm not sure about adding root attributes below, though. This assumes that every user will want svgs to be positioned absolutely. in many cases, such as in buttons, inline-block is more ideal. I feel as though root attributes should be reserved to a per-use-case basis. 

What do you think? Is this global for all/most projects?

Comment (jacobarriola) [inline Gulpfile.js]: That's a great point about the attributes. I'm still pretty fresh in the SVG world, so I'm sure that there are plenty of ways to skin that SVG cat. :smile_cat: I wouldn't want to force developers into a certain corner with styling.

Want me to remove this object and re-commit?

Comment (una) [inline Gulpfile.js]: That would be awesome :smile:

**Comment by jacobarriola** [comment]
@una I removed the root attributes, k. Let me know if I can help in any other way! :v:

**Comment by una** [comment]
:+1:

**Pull request merged**